packages/passport/README.md

```diff
@@ -0,0 +1,84 @@
+# @web3-onboard/passport
```

Comment:
We will also need to add this info around the package to the internal docs site similar to as being done here - https://github.com/blocknative/web3-onboard/pull/2188/files#diff-880dac278a88e1be5c5d7e94dedea953ce24d29259488ff2280d1c754b5f4f1d

Reply:
added